corehq/apps/hqwebapp/models.py

```diff
@@ -67,7 +67,7 @@ class UserAccessLog(models.Model):
 
     user_id = models.CharField(max_length=255, db_index=True)
     action = models.CharField(max_length=20, choices=ACTIONS)
-    ip = models.GenericIPAddressField()
+    ip = models.GenericIPAddressField(blank=True, null=True)
```

Comment:
Unfortunately, yes. From [the GenericIPAddressField docs](https://docs.djangoproject.com/en/3.1/ref/models/fields/#genericipaddressfield): `If you allow for blank values, you have to allow for null values since blank values are stored as null.`

Reply:
oof, that's terrible.